pylearn2/models/dbm/layer.py

```diff
@@ -2486,6 +2486,27 @@ class ConvMaxPool(HiddenLayer):
     .. todo::
 
         WRITEME
+
+    Implements a hidden convolutional layer. The layer lives in a Conv2DSpace.
+
+    Parameters
+    ----------
+    output_channels : WRITEME
+	kernel_rows : WRITEME
+	kernel_cols : WRITEME
+	pool_rows : WRITEME
+	pool_cols : WRITEME
+	layer_name : str
+        Name of the layer
+	center : bool
+        If True, use Gregoire Montavon's centering trick
+	irange : float
+        If specified, initialize the weights in U(-irange, irange)
+	sparse_init : WRITEME
+	scale_by_sharing : WRITEME
+	init_bias : WRITEME
+	border_mode : WRITEME
+	output_axes : WRITEME
```

Comment:
Can you remove the initial "todo: writeme", and get rid of the tabulations and strange indentation?